src/test/ui/async-await/issue-75785-confusing-named-region.stderr

```diff
@@ -1,14 +1,10 @@
 error[E0506]: cannot assign to `*x` because it is borrowed
   --> $DIR/issue-75785-confusing-named-region.rs:9:5
    |
-LL | pub async fn async_fn(x: &mut i32) -> (&i32, &i32) {
```

Comment:
This seems like a regression - the new error message doesn't explain why the assignment is really a problem.

Reply:
I'm not sure I agree with you. The problem is that `x` is assigned to while being immutably borrowed into `y` which is used later. It does not have anything to do with the fact that `x` or `y` appear in the signature, does it?

Reply:
This diff also removed the part where "later use" is shown through